### PR TITLE
🚦HTTP -> HTTPS Redirection for Sparkhacks.

### PR DESCRIPTION
**Pull request by SohamG**

### CODE REVIEW

**Comment by SohamG** [comment]
I fixed the linter but it absolutely hates every yaml file, so it fails.

File: kubernetes/argocd/stacks/traefik/values.yml [hunk 1]
```diff
@@ -13,3 +13,7 @@ tlsStore:
   default:
     defaultCertificate:
       secretName: default-cert
+ports:
+  web:
+    redirectTo:
+      port: websecure
```
Comment (clee231) [inline kubernetes/argocd/stacks/traefik/values.yml:19]: This definitely isn't valid.
These are values passed into the Helm chart which don't call for a `redirectTo` attribute here.

File: kubernetes/argocd/stacks/umpf/umpf.yml [hunk 2]
```diff
@@ -23,6 +23,8 @@ kind: Deployment
 metadata:
   name: umpf
   namespace: umpf
+  annotations:
+    kube-linter.io/ignore-all: "Wtf even is this?"
```
Comment (clee231) [inline kubernetes/argocd/stacks/umpf/umpf.yml:27]: Can we create an issue to fix whatever the linter is complaining about?
Document the errors in the issue, so we know at least what needs to be fixed.

File: kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml [hunk 3]
```diff
@@ -63,6 +63,7 @@ metadata:
 spec:
   entryPoints:
     - websecure
+    - web
```
Comment (clee231) [inline kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml:66]: If you are doing a redirect, then you wouldn't add an entrypoint here. You'd likely use another `IngressRoute` with some redirect logic instead.

**Review by clee231 — CHANGES_REQUESTED** [review]
I'm glad we're actually linting now! Yay! 🎉

File: kubernetes/argocd/stacks/traefik/values.yml (hunk 1 above)
Comment (clee231) [reply kubernetes/argocd/stacks/traefik/values.yml:19]: Sorry, I retract my previous statement on this.
I was looking for a `.Values.ports.web.redirectTo` variable used in the Helm chart, however it that isn't ever used in the Helm chart. 
Discussed on Discord and it was mentioned that the syntax is from the Helm chart values.yaml file itself, so this is probably correct.

```
    # Port Redirections
    # Added in 2.2, you can make permanent redirects via entrypoints.
    # https://docs.traefik.io/routing/entrypoints/#redirection
    # redirectTo: websecure
```

Comment (clee231) [reply kubernetes/argocd/stacks/traefik/values.yml:19]: It's important to note that we currently use the `23.1.0` Helm chart version, which has a slightly different syntax than the latest.
image

We should either just change it to the syntax used in the `23.1.0` chart, or upgrade and make the corresponding changes in the `values.yml`.

Comment (SohamG) [reply kubernetes/argocd/stacks/traefik/values.yml]: Fixed in newest commit

File: kubernetes/argocd/stacks/umpf/umpf.yml (hunk 2 above)
Comment (SohamG) [reply kubernetes/argocd/stacks/umpf/umpf.yml:27]: Uuh, okay. Is it okay if I do that in a new PR?

Comment (clee231) [reply kubernetes/argocd/stacks/umpf/umpf.yml:27]: Most definitely in a different PR. I'm just saying to create a new Github issue so we don't totally lose track of that work.

File: kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml (hunk 3 above)
Comment (clee231) [reply kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml:66]: If we add redirects for the entire `web` entry point, this probably doesn't need to be here unless you still expect to serve the actual website over HTTP.

**Review by clee231 — APPROVED** [review]

**Pull request merged**